app/query_param.go

```diff
@@ -6,8 +6,7 @@ import (
 	"time"
 
 	"github.com/labstack/echo"
-
-	log "github.com/sirupsen/logrus"
+	"github.com/op/go-logging"
```

Comment:
To use go-logging, we need to use its wrapper:

```suggestion
	"github.com/topfreegames/mqtt-history/logger"
```